database/mariadb/README.md

```diff
@@ -171,18 +190,20 @@ Then, exit the container:
 
 We now have a working database, and are ready to start the instance.
 
-The database files are stored on the host under <em>mariadb/db/</em>:
+The database files are stored on the host under `mariadb/db/`:
 ```
 $ singularity instance start \
- -B mariadb/db:/var/lib/mysql \
- -B mariadb/log:/var/log/mysql \
- -B mariadb/run:/var/run/mysqld \
- mariadb.sif mariadb
+-B mariadb/db:/var/lib/mysql \
+-B mariadb/log:/var/log/mysql \
+-B mariadb/run:/var/run/mysqld \
+-B mariadb/etc/:/etc/mysql/mariadb.conf.d \
```

Comment:
```
-B mariadb/etc/:/etc/mysql/mariadb.conf.d \
-B mariadb/conf/:/etc/mysql/conf.d \
```
Is this really needed? I don't require binding the directories under /etc/mysql to run.